packages/components/src/components/as-responsive-content/README.md

```diff
@@ -14,6 +14,38 @@ The component searches for these classes in order to know how many tabs it must
 
 As long as you follow the proper markup, as the `Layout` section shows, you won't have to worry about anything else.
 
+This component replaces `.as-content` HTML element, so it needs to wrap all your children elements to work properly. Here is an example:
```

Comment:
I think this should be with the `>` markdown tag, this way its shown as a ⚠️  in our docs.

> This component replaces `.as-content` HTML element, so it needs to wrap all your children elements to work properly.

Reply:
I don't know if we should put it like a warning 🤔 
It should be clear as long as you read the whole text. I put the ready event text as a warning, because I felt like it was more important.